foursight_core/react/api/auth.py

```diff
@@ -172,6 +212,8 @@ def create_authtoken(self, jwt: str, jwt_expires_at: int, domain: str) -> str:
             "domain": domain,
             "site": app.core.get_site_name()
         }
+        if user_exception:
+            authtoken_decoded["user_exception"] = True
```

Comment:
This ends up in the JWT authtoken cookie (which is used to the the user a more targeted error in the UI), so would rather not weigh it down with an exception message using this; but good point, it would be useful to get a handle on this error/exception message, and somehow perhaps present to the user if desired; will see if I can rework this a bit to do something more useful.